omnigan/trainer.py

```diff
@@ -324,15 +315,30 @@ def run_epoch(self):
             # (batch_domain_0, ..., batch_domain_i)
             # and send it to self.device
             print(
-                "\rEpoch {} batch {} step {}".format(
-                    self.logger.epoch, i, self.logger.global_step
-                )
+                "\rEpoch {} batch {} step {}".format(self.logger.epoch, i, self.logger.global_step)
             )
+
+            """
+            print("**********BATCH INFORMATION********************")
```

Comment:
you and your strings as comments... :)

Reply:
For some reason this is a result of my auto-formatting. And yes, before you ask, I am using black ;)

Reply:
I know you are, I can see it I'm so proud! I don't care about strings as commends, I'm not asking you to change it but I'm not sure how common that is. I guess one argument that could be made is that unlike comments they do take a little memory. But who cares, we're doing deep learning